cdap-app-fabric/src/main/java/co/cask/cdap/app/DefaultAppConfigurer.java

```diff
@@ -64,7 +66,7 @@
 public class DefaultAppConfigurer extends DefaultPluginConfigurer implements ApplicationConfigurer {
   private final ArtifactRepository artifactRepository;
   private final PluginInstantiator pluginInstantiator;
-  private final Id.Artifact artifactId;
+  private final co.cask.cdap.proto.id.ArtifactId artifactId;
```

Comment:
You can remove `co.cask.cdap.api.artifact.ArtifactId` import instead have `co.cask.cdap.proto.id.ArtifactId`, so we don't need to qualify with package.